praw/models/__init__.py

```diff
@@ -15,15 +15,16 @@
 from .reddit.multi import Multireddit  # NOQA
 from .reddit.redditor import Redditor  # NOQA
 from .reddit.submission import Submission  # NOQA
-from .reddit.subreddit import Subreddit  # NOQA
+from .reddit.subreddit import ModmailAction, ModmailConversation, ModmailMessage, Subreddit  # NOQA
```

Comment:
Two questions: Should the `Modmail` class itself continue to live in the `subreddit` module? And what about naming conventions? `reddit.modmail.ModmailAction` seems redundant, but `reddit.modmail.ModmailMessage` looks necessary since `models.Message` already exists.